data_hub_api/docmaps/v2/docmap_typing.py

```diff
@@ -83,10 +83,40 @@
     }
 )
 
+DocmapActorAnonymous = TypedDict(
+    'DocmapActorAnonymous',
+    {
+        'type': str,
+        'name': str
+    }
+)
+
+DocmapAffiliation = TypedDict(
+    'DocmapAffiliation',
+    {
+        'type': str,
+        'name': str,
+        'location': Optional[str]
+    }
+)
+
+DocmapActorEditor = TypedDict(
```

Comment:
Just to consider, whether `DocmapEditorActor` would be better?
The last part would be the main type, `Actor` in this case.
Like `Output` is the main type in `DocmapElifeManuscriptVorOutput`.
(Same for `DocmapActorAnonymous` or `DocmapAnonymousActor`)